source/cl/scripts/cts-3.0-online-extra-linux-riscv.csv

```diff
@@ -0,0 +1,2 @@
+Select,select/test_select -w
+Math,math_brute_force/test_bruteforce -w\[512]
```

Comment:
So we are replacing test_select with `test_select -w`?
what does the \[512] bit do and how does this make it better than "test_bruteforce -w"?

Reply:
It increases the Wimpy Reduction Factor, basically making it even wimpier. Hopefully by enough.

Reply:
why does it need \[ but not \]?

Reply:
I assume GitHub ate your backslash there. Shell quoting rules, `-w[512]` looks for files named `-w5`, `-w1`, `-w2` in the current directory. We don't want that, we want to just pass the literal argument `-w[512]`.